.github/workflows/actions.yml

```diff
@@ -0,0 +1,43 @@
+name: Lint
```

Comment:
Why name is Lint, but in the pipeline we also have tests

Reply:
done